app/Livewire/GlobalSearch.php

```
@@ -419,22 +599,728 @@ private function loadSearchableItems()
+            $creatableSuggestions = collect($this->creatableItems)
+                ->filter(function ($item) use ($query) {
+                    $searchText = strtolower($item['name'].' '.$item['description'].' '.($item['type'] ?? ''));
+
+                    // Use word boundary matching to avoid substring matches (e.g., "wordpress" shouldn't match "classicpress")
+                    return preg_match('/\b'.preg_quote($query, '/').'/i', $searchText);
+                })
+                ->map(function ($item) use ($query) {
+                    // Calculate match priority: name > type > description
+                    $name = strtolower($item['name']);
+                    $type = strtolower($item['type'] ?? '');
+                    $description = strtolower($item['description']);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$description'. (undefined)

(UnusedLocalVariable)